app/models/team.rb

```diff
@@ -3,4 +3,12 @@ class Team < ActiveRecord::Base
   has_many :income
   has_many :bonus_credit
   has_many :public_relation
+
+  def self.countries
+    self.pluck(:team_name).delete_if {|x| x == "Aliens"}
+  end
+
+  def self.all_minus_aliens
+    self.all.where.not(team_name: "Aliens")
```

Comment:
Yeah, I want to fix this in the future mostly with a hidden flag like we talked about.  Can't just override self.all though because we use it to get the list of who you can communicate with in Api_controller.